extension/src/popup/components/account/AccountHeader/index.tsx

```diff
@@ -24,14 +25,14 @@ import IconCube from "popup/assets/icon-cube.svg";
 import "./styles.scss";
 
 interface AccountHeaderProps {
-  accountDropDownRef: React.RefObject<HTMLDivElement>;
+  // accountDropDownRef: React.RefObject<HTMLDivElement>;
   allAccounts: Array<Account>;
   currentAccountName: string;
   publicKey: string;
 }
 
 export const AccountHeader = ({
-  accountDropDownRef,
+  // accountDropDownRef,
```

Comment:
What is this `ref` used for? I noticed we use refs in a few other places.

Reply:
hmm I don't see this used anywhere, maybe leftover from a refactor that removed the need for it?

Reply:
Hm, we usually use refs to dynamically set modal/slide up heights and positioning. I believe at one point this ref was used to show the dropdown of accounts on the main screen. But looking at this code, it looks like we refactored this at some point and this is not needed anymore. I'll QA this to make sure we can safely remove